gotest/main.go

```diff
@@ -1,5 +1,6 @@
 package main
 import ("fmt")
 func main(){
-	fmt.Println("rahulpc")
+	fmt.Println("rahul___pc")
```

Comment:
change thi s line to rahulpc Println as the original.

Reply:
ok i will do it creating new branch.